namespace cpp generated

include "shared.thrift"

const shared.Text SERVICE1_NAME = "Service1"
const shared.Text SERVICE2_NAME = "Service2"
